src/cmd/gen-sops.ts

```diff
@@ -77,6 +77,7 @@ export const genSops = async (): Promise<void> => {
     if (serviceKeyJson) {
       debug.log('Creating gcp-key.json for vscode.')
       writeFileSync(`${env.ENV_DIR}/gcp-key.json`, JSON.stringify(serviceKeyJson))
+      writeFileSync(`${env.ENV_DIR}/.secrets`, `GCLOUD_SERVICE_KEY='${JSON.stringify(serviceKeyJson)}'`, { flag: 'a' })
```

Comment:
Is this not resulting in many lines appended every time? If any secrets should get written to `.secrets` those should be complete (as there are many more) and done in a purpose built function imo, so not here.

Reply:
@Dunky13 could you refer to this comment?

Reply:
It is indeed not an ideal solution.
Better way:
```
1. Does .secrets exist?
1.a: If yes, does it contain GCLOUD_SERVICE_KEY
1.a.a If yes, do nothing - or replace
1.a.a If no, append
1.a If no, create & append
```

Reply:
like I said this may involve many more than just that var...lets remove it here for now

Reply:
please remove this line

Reply:
```suggestion
```